scripter.py

```diff
@@ -43,6 +43,17 @@ def __linux_create_wallpaper_script(pokemon):
     if "gnome" in os.environ.get("GDMSESSION"):
         fmt = 'gsettings set org.gnome.desktop.background picture-uri "file://{}"'
         return fmt.format(pokemon.get_path())
+    
+    if "xubuntu" in os.environ.get("GDMSESSION"):
```

Comment:
I don't have a machine to test that. But you got a point. 

If someone can fetch me whats the value of `GDMSESSION` env on a system thats not xubuntu I'd totally appreciate it. Anyway as soon as I can fetch one I'll test the env to fix the scripter.